src/python/grpcio/grpc/experimental/aio/_base_server.py

```diff
@@ -133,6 +133,15 @@ def add_secure_port(self, address: str,
 class ServicerContext(Generic[RequestType, ResponseType], abc.ABC):
     """A context object passed to method implementations."""
 
+    @abc.abstractmethod
+    def add_done_callback(self, callback: DoneCallbackType) -> None:
+        """Registers a callback to be called on RPC termination.
```

Comment:
Let's be more explicit here. The main use case for this method is cancellation. Let's enumerate the ways in which an RPC can terminate (cancellation, successful completion, etc.)